packages/create/src/index.ts

```diff
@@ -215,15 +218,15 @@ async function init() {
 
   console.log(`\nScaffolding project in ${root}...`)
 
-  runCommand(
-    `npm create vite@latest ${targetDir} -- --template react-ts`,
-    cwd,
-    true,
-  )
+  execCommandExitOnError(`npm create vite@latest "${targetDir}" -- --template react-ts`)
```

Comment:
You might be able to code with a then or a delay, but the core reason for retry is to avoid any I/O delays with the OS that could happen when the node asks for a file but the OS has not fully registered filesystem changes. For example, if you are working within a virtual or symlinked folder, there can be a delay between a written file and when you can read it, and we want to handle edge cases to make the program more robust.